.github/workflows/main.yml

```diff
@@ -43,6 +47,8 @@ jobs:
       - name: Upload coverage data
         continue-on-error: ${{ matrix.continue || false }}
         uses: codecov/codecov-action@v1
+        with:
+          name: os:${{ matrix.os }} py:${{ matrix.python }}
```

Comment:
Where can I see the different report names on codecov?

Reply:
https://codecov.io/gh/streamlink/streamlink/commit/6ef3bbc99eee5abbf3e049c5e8f01326b456a7c4/build